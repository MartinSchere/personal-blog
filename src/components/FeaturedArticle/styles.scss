.post-list-item {
  h4 {
    font-weight: 600;
  }
  .article-excerpt {
    display: none;

    @include respond-to("medium") {
      display: block;
    }
  }
}
