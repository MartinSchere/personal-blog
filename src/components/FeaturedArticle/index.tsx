import React from 'react';
import "./styles.scss";

type FeaturedArticleProps = {

}

const FeaturedArticle = (props:FeaturedArticleProps) => {
return (
<div>
  FeaturedArticle
</div>
);
};

export default FeaturedArticle;