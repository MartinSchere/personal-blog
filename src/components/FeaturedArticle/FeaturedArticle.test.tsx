import React from 'react';
import FeaturedArticle from './index';
import renderer from 'react-test-renderer';

describe('FeaturedArticle', () => {
it('renders without error', () => {
const tree = renderer
.create(<FeaturedArticle></FeaturedArticle>)
.toJSON();
expect(tree).toMatchSnapshot();
});
});