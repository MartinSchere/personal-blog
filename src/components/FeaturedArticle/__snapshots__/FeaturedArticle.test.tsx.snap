// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeaturedArticle renders without error 1`] = `
<div>
  FeaturedArticle
</div>
`;
